package.json

```diff
@@ -14,16 +19,17 @@
     "@babel/node": "^7.8.3",
     "@babel/polyfill": "^7.8.3",
     "@babel/preset-env": "^7.8.3",
-    "@babel/register": "^7.8.3",
+    "babel-plugin-add-module-exports": "^1.0.2",
     "chai": "^4.2.0",
     "chai-http": "^4.3.0",
-    "coveralls": "^3.0.9",
+    "cross-env": "^6.0.3",
     "eslint": "^6.8.0",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-import": "^2.20.0",
+    "nodemon": "^1.19.4",
     "mocha": "^7.0.0",
-    "nyc": "^15.0.0",
-    "nodemon": "^1.19.4"
+    "nyc": "^15.0.0"
+    "coveralls": "^3.0.9"
```

Comment:
There's is an error here, kindly fix this.
Thank you.

Reply:
Fixed

Reply:
I think the error is still there, please fix it